pkgs/top-level/all-packages.nix

```diff
@@ -32978,6 +32978,8 @@ with pkgs;
 
   multilockscreen = callPackage ../misc/screensavers/multilockscreen { };
 
+  i3bar-river = callPackage ../applications/misc/i3bar-river { };
```

Comment:
can you use the new by-name structure?

https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#name-based-package-directories

Reply:
Done, could you take another look?